### PR TITLE
MAKE K8S SJ MANAGER RUN FASTER: As Emmanuel, I want the /parsers page to load much faster, so that I dont get frustrated with general slowness and have to wait 9 seconds for certain pages to load

### PR DESCRIPTION
**Pull request by Reinkaos**
**Acceptance Criteria**

- SJ manager's overall page load speed is improved
- This page loads in 4 seconds or less
http://manager.digitalnz.org/parsers

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -47,6 +47,7 @@ gem 'foundation-rails'
 gem 'jquery-datatables-rails', git: 'https://github.com/rweng/jquery-datatables-rails.git'
 gem 'activeresource-response'
 gem 'puma'
+gem 'server_timing'
```
Comment (motizuki) [inline Gemfile]: do we need this in prod?

File: app/controllers/parsers_controller.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@ class ParsersController < ApplicationController
   respond_to :json, :html
 
   def index
-    @parsers = Parser.all
+    @parsers = Parser.only(:name, :strategy, :source, :data_type, :created_at, :updated_at, :last_editor).all.to_a
```
Comment (motizuki) [inline app/controllers/parsers_controller.rb]: 🥇

File: app/views/parsers/index.html.erb [hunk 3]
```diff
@@ -24,8 +24,8 @@
           <td><%= parser.strategy %></td>
           <td><%= link_to parser.source.partner.name, edit_partner_path(parser.source.partner) %></td>
           <td><%= link_to parser.source.source_id, edit_source_path(parser.source) %></td>
-          <td><%= parser.versions.last.try(:created_at).try(:localtime)&.to_formatted_s(:long) %></td>
-          <td><%= parser.last_edited_by %></td>
+          <td><%= parser.try(:updated_at).try(:localtime)&.to_formatted_s(:long) %></td>
```
Comment (invke) [inline app/views/parsers/index.html.erb]: Just double checking - but does the table head for this column make sense with the changes / not need to be renamed or anything?

File: Gemfile (hunk 1 above)
Comment (Reinkaos) [reply Gemfile]: Yeah I think it's useful for Dan to see.

File: app/controllers/parsers_controller.rb (hunk 2 above)
Comment (Reinkaos) [reply app/controllers/parsers_controller.rb]: The magic haha

File: app/views/parsers/index.html.erb (hunk 3 above)
Comment (Reinkaos) [reply app/views/parsers/index.html.erb]: Na it's the same value :).

**Pull request merged**